test/Utils/Flex/data/cloudsql-describe-result

```diff
@@ -0,0 +1,42 @@
+{
+  "automaticScaling": {
+    "coolDownPeriod": "120s",
+    "cpuUtilization": {
+      "targetUtilization": 0.5
+    },
+    "maxTotalInstances": 20,
+    "minTotalInstances": 2
+  },
+  "betaSettings": {
+    "cloud_sql_instances": "my-project:us-central1:my-instance",
+    "has_docker_image": "true",
+    "module_yaml_path": "app.yaml",
```

Comment:
If the name of the `app.yaml` is available, it might be possible to attempt to read the cloudsql instances from that `app.yaml`.

Reply:
See above comment :)